kube/pod.go

```diff
@@ -204,9 +204,19 @@ func getEnvVars(role *model.Role, defaults map[string]string, secrets SecretRefM
 			continue
 		}
 
-		ok, stringifiedValue := configValue(config, defaults)
-		if !ok {
-			continue
+		var stringifiedValue string
+		if createHelmChart {
+			required := ""
+			if config.Required {
+				required = fmt.Sprintf(`required "A valid .Values.env.%s is required" `, config.Name)
```

Comment:
Maybe I'm missing something - can't the error be `UAA_PORT configuration missing.`?
I know the full name is longer, but the operator doesn't care.